modules/depend_on_what_you_use/0.1.0/MODULE.bazel

```diff
@@ -0,0 +1,64 @@
+module(
+    name = "depend_on_what_you_use",
+    # Keep in sync with setup_step_2.bzl
+    bazel_compatibility = [">=5.4.0"],
+    compatibility_level = 0,
+  version = "0.1.0",
+)
+
+bazel_dep(
+    name = "bazel_skylib",
+    # Keep in sync with //third_party/dependencies.bzl
```

Comment:
Since `MODULE.bazel` uses Minimum Version Selection, it's generally recommended to keep the versions of dependencies as low as possible. (With `WORKSPACE`, the opposite was true as your indirect deps may shadow those of other dependencies.) Bumping the versions in lockstep with WORKSPACE deps thus shouldn't be necessary. Instead, you can bump when you need a new feature.